kaggle_environments/envs/lux_ai_2021/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script>
+      /*window.kaggle*/
+    </script>
+    <meta charset="UTF-8" />
+    <title>Lux AI Challenge Viewer</title>
+    <meta
+      name="viewport"
+      content="width=device-width,initial-scale=1,user-scalable=no"
+    />
+    <link rel="icon" href="https://2021vis.lux-ai.org/icon.png" />
+    <meta name="HandheldFriendly" content="True" />
+    <meta name="MobileOptimized" content="320" />
+    <meta http-equiv="cleartype" content="on" />
+    <meta name="format-detection" content="telephone=no" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <style>
+      html,
+      body {
+        margin: 0;
+        padding: 0;
+      }
+    </style>
+  </head>
+  <body>
+    <div id="root"></div>
+    <!-- <script src="kaggle_environments/envs/lux_ai_2021/dist/vendors~app~phaser.js"></script>
+    <script src="kaggle_environments/envs/lux_ai_2021/dist/vendors~app.js"></script>
+    <script src="kaggle_environments/envs/lux_ai_2021/dist/app.js"></script>
+    <script src="kaggle_environments/envs/lux_ai_2021/dist/phaser.js"></script> -->
+    <script src="https://2021vis.lux-ai.org/vendors~app~phaser.js"></script>
```

Comment:
We'll need to be able to reference these locally rather than from an absolute URL.

Reply:
This is possible. I originally kept it as an absolute URL mostly because I didn't want to have to update kaggle envs every time the visualizer receives an update (that doesn't affect game logic). I will change this to be local instead.

Moreover, if I make it local, how do I know this will work in that embedded viewer on the kaggle website? I can get it working locally so that when render mode is HTML, it will produce a working HTML file but how will it work online?

Also maybe this is an indication that we should discuss about how this will be updated as the competition progresses? I expect there may be the occasional game logic or parameter change in the competition and updates to the visualizer.

Reply:
Actually follow up question, why do these need to be local? I see that in `static/player.html` there are files that are fetched from elsewhere:

https://github.com/Kaggle/kaggle-environments/blob/24a2cea763fe5d68a9f0f4c90b66d13801650316/kaggle_environments/static/player.html#L23

https://github.com/Kaggle/kaggle-environments/blob/24a2cea763fe5d68a9f0f4c90b66d13801650316/kaggle_environments/static/player.html#L51

Can we not do the same?

Reply:
Could we obtain phaser through an official cdn? If so I'm fine with that.